meta-oe/recipes-devtools/guider/guider_3.9.9.bb

```diff
@@ -1,14 +1,14 @@
-SUMMARY = "performance analyzer"
+SUMMARY = "unified performance analyzer"
 HOMEPAGE = "https://github.com/iipeace/guider"
 BUGTRACKER = "https://github.com/iipeace/guider/issues"
 
 LICENSE = "GPL-2.0-or-later"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=2c1c00f9d3ed9e24fa69b932b7e7aff2"
 
-PV = "3.9.8+git"
+PV = "3.9.9+git"
 
 SRC_URI = "git://github.com/iipeace/${BPN};branch=master;protocol=https"
```

Comment:
Latest OE now benefits if we define `tag=` in the SRC_URI, so please add it along while you are here. e.g to in this recipe

```
SRC_URI = "git://github.com/iipeace/${BPN};branch=master;protocol=https;tag=version/3.9.9"
```

Reply:
I updated SRC_URI.